lib/app/app.dart

```diff
@@ -1,5 +1,5 @@
 import 'package:flutter/material.dart';
-import 'package:to_do_list_kamilg/feature/view/tasks_page.dart';
+import 'package:to_do_list_kamilg/home/view/tasks_page.dart';
```

Comment:
use barrel files instead

Reply:
you did not change it

Reply:
[Create barel function](https://github.com/kamilgadek/to_do_list_kamilg/pull/21/commits/66a968ae74aefeed372fc3d656795987df4a8e3b)